commit 9d109b568bbbdee069bb5dfcbca3c259a74e98a8
Author: Denys Vlasenko <vda.linux@googlemail.com>
Date:   Wed Mar 3 11:45:33 2010 +0100

    abrt-debuginfo-install: remove -R2 from yum! it's not "anti-yum-lock" option!
    
    Signed-off-by: Denys Vlasenko <vda.linux@googlemail.com>

diff --git a/src/Daemon/abrt-debuginfo-install b/src/Daemon/abrt-debuginfo-install
index 3bb9c41..1b6af74 100755
--- a/src/Daemon/abrt-debuginfo-install
+++ b/src/Daemon/abrt-debuginfo-install
@@ -146,11 +148,12 @@ print_package_names() {
     # when we look for debuginfo we need only -debuginfo* repos, so we can disable the rest and thus make it faster
     # also we want only fedora repositories, because abrt won't work for other packages anyway
     # --showduplicates: do not just show the latest package
-    # -R2: wait two minutes max (hopefully this prevents infinite hang on yum lock)
-    local cmd="yum $yumopts '--disablerepo=*' '--enablerepo=fedora-debuginfo*' '--enablerepo=updates-debuginfo*' --showduplicates -R2 --quiet provides $missing_debuginfo_files"
+    # (tried to use -R2 to abort on stuck yum lock but -R is not about that)
+    local cmd="yum $yumopts '--disablerepo=*' '--enablerepo=fedora-debuginfo*' '--enablerepo=updates-debuginfo*' --showduplicates --quiet provides $missing_debuginfo_files"
     echo "$cmd" >"yum_provides.$1.OUT"
+    $debug && echo "Running: $cmd" >&2
     # eval is needed to strip away ''s; cant remove them above and just use
-    # $cmd, that would perform globbing on '*'
+    # unquoted $cmd, that would perform globbing on '*'
     local yum_provides_OUT="`eval $cmd 2>&1`"
     local err=$?
     printf "%s\nyum exitcode:%s\n" "$yum_provides_OUT" $err >>"yum_provides.$1.OUT"
